ci/authn-gcp/get_gcp_id_tokens.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
who calls this file `ci/authn-gcp/get_gcp_id_tokens.sh`?

Reply:
dev/start